### PR TITLE
[Documentation] Documentation Consistency and README

### PR DESCRIPTION
**Pull request by harshsbajwa**
Updated documentation and privatized attributes in Item,
Updated affected files (Customization and ItemTest)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,11 @@
...
 It also contains workflow documents that give instructions on how to manage your Github repository and how to use Github Projects for efficient collaboration.
 
-## Checklist For Your Project
-- [ ] Verify the correct settings for your project repository
-- [ ] Set up Github Projects
-- [ ] Create the implementation plan using issues and Github Projects
-- [ ] Create deveopment branches for your features
-- [ ] Use pull requests to merge finished features into main branch
-- [ ] Conduct code reviews
+My Dog Ate My Homework! (MDAMH) is a task manager with a twist. You begin by choosing a pet, who accompanies you as you accomplish various tasks. Win points to customize your pet, but be careful! If you miss a task deadline your pet will be damaged, requiring you to heal your pet. Miss too many tasks and your pet will die :(
```
Comment (e-laflamme) [inline README.md:8]: We could also add a list of implemented and planned features to the doc in the future

**Review by e-laflamme — APPROVED** [review]
Looks good! Nice documentation

File: README.md [hunk 2]
```diff
@@ -1,39 +1,25 @@
...
-All tests should pass. Your team can remove this sample of how testing works once you start adding your project code to the repo.
-
-Moving forward, we expect you to maintain this project structure. You *should* use Gradle as the build environment, but it is fine if your team prefers to use something else -- just remove the gradle files and push your preferred project setup. Assuming you stick with Gradle, your source code should go into `src/main/java` (you can keep creating more subdirectories as per your project requirement). Every source class can auto-generate a test file for you. For example, open HelloWorld.java file and click on the `HelloWorld` variable as shown in the image below. You should see an option `Generate` and on clicking this your should see an option `Test`. Clicking on this will generate a JUnit test file for `HelloWorld` class. This was used to generate the `HelloWorldTest`.
-
-![image](https://user-images.githubusercontent.com/5333020/196066655-d3c97bf4-fdbd-46b0-b6ae-aeb8dbcf351d.png)
-
-You can create another simple class and try generating a test for this class.
+## How to Run
```
Comment (XinyingXu) [inline README.md:25]: This ReadMe file looks great! We can try to figure out how to run the code at a later stage.

**Review by XinyingXu — APPROVED** [review]

**Review by serenahonekin — APPROVED** [review]
The wording is clear and stylistic!

**Pull request merged**